code/datums/credit.dm

```diff
@@ -0,0 +1,17 @@
+/datum/credit
+	var/original_owner_name = "None"
+	var/pin = "0000"
+	var/acc_number = "12345"
+	var/balance = 0
+	var/special = 0
+
+/datum/credit/New(var/special = 0)
+	. = ..()
+	pin = "[rand(1000,9999)]"
```

Comment:
:-1:

Remove PINs, make your credit card your ID.

Reply:
also Initialize rather than New

Reply:
@Iamgoofball what if you lose it? or switch to an agent ID? I also plan to implement the account into roundstart IDs
@praisenarsie datums dont have Initialize last i checked

Reply:
Just make "e-wallets" that are in PDAs and can be downloaded via tablets. The e-wallets generate a token for transactions and act like bitcoin wallets.

Reply:
@MoreRobustThanYou That's life, don't lose your ID or your credit card.

Reply:
@Iamgoofball or you get pda bombed?

Reply:
rip life savings